doc/radosgw/multisite.rst

```diff
@@ -40,25 +40,28 @@ non-master zones. Following are the basic terminologies that would be used:
 About this guide
 ================
 
-In this guide we create a single zone group, with three separate zones, which
-actively sync data between them. For the purpose of the guide, we create 2 zones
-in the same cluster, and the third zone in a different cluster. There is no sync
-agent involved for mirroring data changes between the RadosGWs and this allows
-for a much simpler configuration scheme and active-active configurations. Please
-note that metadata operations such as creating a user would still need to go
-through the master zone, however data operations such as creation of buckets
-objects etc. can be handled by any of the zones.
+In this guide we use 2 Ceph clusters, and create a single zonegroup,
+with three zones, which sync data between them actively. For the
+purpose of the guide, we'll create 2 zones in the same cluster, and 1
+zone in a different cluster. There is no sync agent involved for
+mirroring data changes between the Ceph Object Gateway instances and
+this allows for a much simpler configuration scheme and active-active
+configurations. Please note that metadata operations such as creating
+a user would still need to go through the master zone, however data
+operations such as creation of buckets objects can be handled by any
+of the zones.
 
 System Keys
 -----------
 
-While configuring zones, RadosGW, expects creation of zone a system user, with
-an S3-like access and secret keys. This allows another radosgw instance to pull
-the configuration remotely, with the access and secret keys. For making
-scripting and use of configuration management tools easier, it would be easier
-to generate the access-key and secret-key before hand and use these. For the
-purpose of the guide we assume an access key and secret key are set in the
-environment. For eg., something like::
+While configuring zones, Ceph Object Gateway instance, expects
+creation of zone a system user, with a S3-like access and secret keys.
```

Comment:
@mikulely just a couple of comments on the systemd commands to start rgw, (since you're touching most of the doc, we'll correct those as well)